partcad-cli/src/partcad_cli/click/commands/inspect.py

```
@@ -0,0 +1,109 @@
+    if assembly:
+        obj = ctx.get_assembly(path, params=params)
+    elif interface:
+        obj = ctx.get_interface(path)
+    elif sketch:
+        obj = ctx.get_sketch(path, params=params)
+    else:
+        obj = ctx.get_part(path, params=params)
+
+    if obj is None:
+        if package is None:
+            pc_logging.error("Object %s not found" % object)
```

Ruff (0.8.0)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)